packages/langium/src/parser/langium-parser.ts

```diff
@@ -213,7 +213,7 @@ export class LangiumParser extends AbstractLangiumParser {
             cstNode = this.nodeBuilder.buildCompositeNode(feature);
         }
         const subruleResult = this.wrapper.wrapSubrule(idx, rule, args) as any;
-        if (!this.isRecording() && cstNode && cstNode.length > 0) {
+        if (!this.isRecording() && cstNode && cstNode.content.length > 0) {
```

Comment:
I didn't get the intention behind this change. Instead of checking the containing text length we now check if there is more than one child node, even if they are probably empty? It raises a question what will happen with an empty range when later building a reference with `this.linker.buildReference(...)`